### ISSUE TITLE
Batch Norm after activation

### ISSUE THREAD

**Issue by n3011** [alrojo/lasagne_residual_network#1]
In original paper, normalization was used before activation, but it seem you have used after activation.
@alrojo Am I wrong?

**Comment by alrojo** [comment]
I assume you are refering to:
https://github.com/alrojo/lasagne_residual_network/blob/master/Deep_Residual_Network_mnist.py#L94

It might seem like that is the case, but if you take a look at the BatchNormlizationLayer.py file:
https://github.com/alrojo/lasagne_residual_network/blob/master/BatchNormLayer.py#L320

The batch_norm layer that is used does the following:
"Apply batch normalization to an existing layer. This is a convenience
    function modifying an existing layer to include batch normalization: It
    will steal the layer's nonlinearity if there is one (effectively
    introducing the normalization right before the nonlinearity), remove
    the layer's bias if there is one (because it would be redundant), and add
    a :class:`BatchNormLayer` and :class:`NonlinearityLayer` on top."

Hope that helps!

**Comment by n3011** [comment]
Thanks! I got it now.
